product-service/src/main/java/com/extremecoder/productservice/utils/UrlConstraint.java

```diff
@@ -0,0 +1,17 @@
+package com.extremecoder.productservice.utils;
+
+public final class UrlConstraint {
+
+    private UrlConstraint() {
+    }
+
+    private static final String API = "/api";
```

Comment:
Do we really need this variable declaration here? We are concatenating this variable only in line number 12. Other than that it's not used anywhere. So i think it's redundant to declare and initialize. Instead we can directly add the `/api` at line no 12.

Reply:
Removed this line and directly add this.

Reply:
Thanks bhai.